NuGet.sln

```diff
@@ -240,6 +240,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NuGet.VisualStudio.Internal
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NuGet.Tools.Test", "test\NuGet.Clients.Tests\NuGet.Tools.Test\NuGet.Tools.Test.csproj", "{FC1F1CF8-4902-46F0-929F-204E5B1B6D53}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NuGet.Signing.CrossFramework.Test", "test\NuGet.Core.FuncTests\NuGet.Signing.CrossFramework.Test\NuGet.Signing.CrossFramework.Test.csproj", "{3CAE192D-FF86-4A51-8936-E4BFFC192C94}"
```

Comment:
The CrossFrameworkVerificationTest targets net5.0 and net472 TFMs.
For net5.0 TFM, the tests will sign with .net5.0 signing APIs and invoke NuGet.exe verify command(net472) to verify the signatures.
For net472 TFM, the tests will sign with .net472 signing APIs and invoke dotnet.exe nuget verify command(net5.0) to verify the signatures. 
If I put it under `test\NuGet.Clients.FuncTests\`, it will only targets net472 then. So I guess I'd better to put it under `src\NuGet.Core`, right?

Reply:
> If I put it under test\NuGet.Clients.FuncTests\, it will only targets net472 then.

The project targets whatever TFMs is in the csproj's `<TargetFrameworks>` property. The folder doesn't affect TFM, as far as I'm aware.

Reply:
Yes, you're right, the folder doesn't affect TFM.
While I think it's kind of difficult to determine which folder it should go under.
The CrossFrameworkVerificationTest is using signing APIs from core(`.\src\NuGet.Core\NuGet.Packaging`) to sign the packages, then run NuGet.exe(in core code path) or dotnet.exe(in full framework code path) to verify them. The dotnet.exe is patched and it's like dotnet.integration.tests.
So I'm not sure if I should create a new folder, or put it under `NuGet.Client.FuncTests`, or `NuGet.Core.FuncTests`.